ProcessMaker/modeler
# FOUR-8725 design and move the autovalidate control

## Issue & Reproduction Steps

Expected behavior: 
Improve the auto-validate control design and move it to the top rail

Actual behavior: 
N/A

## Solution
Re-design and implement the functionalities to the new auto-validate control

## How to Test
Test the steps above

1. Open a BPMN process
2. It should display the new auto-validate control in the top rail of the screen
3. Click on the new auto-validate control in the top rail
4. It should display the issue button next to the auto-validate control
5. Click on this button to show the detailed panel

## Related Tickets & Packages
[FOUR-8725](https://processmaker.atlassian.net/browse/FOUR-8725)

## Code Review Checklist


<sup> Created from VS Code using [CodeStream](https://codestream.com/?utm_source=cs&utm_medium=pr&utm_campaign=github*com)</sup>

[FOUR-8725]: https://processmaker.atlassian.net/browse/FOUR-8725?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
